components/identities/availableIdentities.tsx

```
@@ -29,11 +29,21 @@ import IdentitiesSkeleton from "./skeletons/identitiesSkeleton";
 import DomainExpiredModal from "@/components/UI/domainExpiredModal";
+import { useIdentityRefresh } from "../../hooks/useIdentityRefresh";
 
 const AvailableIdentities = ({ tokenId }: { tokenId: string }) => {
   const router = useRouter();
   const { address } = useAccount();
-  //const tokenId: string = router.query.tokenId as string;
+  
+  let registerRefreshCallback: ((tokenId: string, callback: () => void) => void) | null = null;
+  let unregisterRefreshCallback: ((tokenId: string) => void) | null = null;
+  try {
+    const context = useIdentityRefresh();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)